<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1072</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1072</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>HyperMail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icono.ico</normaloff>icono.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#fondo{
background:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(90, 86, 158, 255), stop:1 rgba(14, 25, 7, 255))


}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="fondo" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-11</y>
      <width>1091</width>
      <height>631</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1091</width>
      <height>631</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1091</width>
      <height>631</height>
     </size>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1081</width>
       <height>611</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1021</width>
       <height>581</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2000</width>
       <height>2000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">fondo{background:rgb(170, 255, 127)}
QWidget#smtp,QWidget#pop3{
background-color:qlineargradient(spread:pad, x1:0.324, y1:0.431591, x2:1, y2:0, stop:0 rgba(18, 32, 32, 255), stop:1 rgba(91, 128, 151, 255));
}
QWidget#config{
background-color:qlineargradient(spread:pad, x1:0.341045, y1:0.523, x2:1, y2:0, stop:0 rgba(18, 32, 32, 255), stop:1 rgba(70, 92, 129, 255));
}

QPushButton{
background:rgb(40, 97, 76);
}

QLabel,QLineEdit,QTextEdit{
background:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(27, 55, 38, 205), stop:0.397727 rgba(255, 255, 255, 0));
color:rgb(255, 255, 255);
}
QLineEdit{
background:qlineargradient(spread:pad, x1:1, y1:0.972, x2:1, y2:0.546, stop:0.232955 rgba(0, 102, 160, 171), stop:0.25 rgba(255, 255, 255, 0));
border:transparent
}
QPushButton{
background:rgb(40, 97, 76);
color:rgb(255, 255, 255);
}
QLabel{
background:transparent
}

QPushButton:hover {
background:rgb(255, 255, 255);
color:rgb(47, 47, 47)
 }
#outputSMTP{
background:qlineargradient(spread:pad, x1:1, y1:0.915, x2:1, y2:0, stop:0 rgba(51, 48, 75, 255), stop:1 rgba(255, 255, 255, 0));
color:rgb(255, 247, 183);
border: 20px;
}
QTextEdit#textMensajeFinal{
background:qlineargradient(spread:repeat, x1:0.484, y1:0.682, x2:0.443136, y2:0.716, stop:0.0909091 rgba(0, 0, 0, 0), stop:0.352273 rgba(33, 68, 77, 149))
}
QListWidget{
color:rgb(219, 255, 207);
background:qlineargradient(spread:pad, x1:1, y1:0.915, x2:1, y2:0, stop:0 rgba(45, 61, 31, 149), stop:1 rgba(88, 88, 88, 46));

border-bottom: 4px solid rgb(148, 156, 125);
border-top:transparent;

}
#label_8{
color: rgb(255, 255, 127)
}
#asdf{
background:qlineargradient(spread:pad, x1:1, y1:0.915, x2:1, y2:0, stop:0 rgba(22, 33, 46, 185), stop:1 rgba(255, 255, 255, 0))
}
#textTotal{
color:rgb(255, 255, 127)
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="smtp">
      <attribute name="title">
       <string>   Enviar Correo   </string>
      </attribute>
      <widget class="QLineEdit" name="textSubject">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>50</y>
         <width>421</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Ingresa titulo de mensaje...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textFrom">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>130</y>
         <width>421</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Ingresa origen...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textTo">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>220</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Ingresa destinatarios, separalos por coma ...</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textMessage">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>290</y>
         <width>441</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="placeholderText">
        <string>Ingresa mensaje...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="botonSend">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>530</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelModo_2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>30</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelModo_3">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>110</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelModo_4">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>190</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelModo_5">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>260</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="outputSMTP">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>481</width>
         <height>471</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>10</y>
         <width>20</width>
         <height>511</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="botonCleanSMTP">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>510</y>
         <width>481</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Limpiar Pantalla</string>
       </property>
      </widget>
      <widget class="QLabel" name="zzz">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>331</width>
         <height>271</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="pixmap">
        <pixmap>gatoUno.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>zzz</zorder>
      <zorder>textSubject</zorder>
      <zorder>textFrom</zorder>
      <zorder>textTo</zorder>
      <zorder>textMessage</zorder>
      <zorder>botonSend</zorder>
      <zorder>labelModo_2</zorder>
      <zorder>labelModo_3</zorder>
      <zorder>labelModo_4</zorder>
      <zorder>labelModo_5</zorder>
      <zorder>outputSMTP</zorder>
      <zorder>line</zorder>
      <zorder>botonCleanSMTP</zorder>
     </widget>
     <widget class="QWidget" name="pop3">
      <attribute name="title">
       <string>   Revisar Buzon   </string>
      </attribute>
      <widget class="QLineEdit" name="textUser">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Ingresa nombre de usuario</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textPassword">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Ingresa clave de acceso</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textMensajeFinal">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>641</width>
         <height>241</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Completa los datos de acceso...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="botonLogin">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="botonDelete">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>530</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Delete Message</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>641</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Buzon</string>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>0</y>
         <width>20</width>
         <height>521</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QListWidget" name="list">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>31</y>
         <width>361</width>
         <height>491</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="botonDeleteAll">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>540</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>DELETE ALL</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>301</width>
         <height>171</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>451</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>gatoB.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>usuario.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="textTotal">
       <property name="geometry">
        <rect>
         <x>856</x>
         <y>533</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>250</y>
         <width>221</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>1uHg.gif</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label_11</zorder>
      <zorder>label_27</zorder>
      <zorder>frame</zorder>
      <zorder>textMensajeFinal</zorder>
      <zorder>botonDelete</zorder>
      <zorder>line_2</zorder>
      <zorder>label_3</zorder>
      <zorder>line_5</zorder>
      <zorder>list</zorder>
      <zorder>botonDeleteAll</zorder>
      <zorder>textPassword</zorder>
      <zorder>label_2</zorder>
      <zorder>label</zorder>
      <zorder>botonLogin</zorder>
      <zorder>textUser</zorder>
      <zorder>label_13</zorder>
      <zorder>textTotal</zorder>
     </widget>
     <widget class="QWidget" name="config">
      <attribute name="title">
       <string>   Configuracion   </string>
      </attribute>
      <widget class="QLineEdit" name="TextMailFrom">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>150</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Small Fonts</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>&lt;root@dns.inf3340.inf&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MAIL FROM:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>210</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>RCPT:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>550</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>HyperMail version 0.1        AlonsoInc © 2021</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>281</width>
         <height>531</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>zelda.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>270</y>
         <width>291</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>z1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>390</y>
         <width>351</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>z2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>300</y>
         <width>351</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>z3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="asdf" native="true">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>230</y>
         <width>1091</width>
         <height>361</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="TextRCPT">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>0</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Small Fonts</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>&lt;usuario@dns.inf3340.inf&gt;</string>
        </property>
       </widget>
      </widget>
      <zorder>label_9</zorder>
      <zorder>label_12</zorder>
      <zorder>label_10</zorder>
      <zorder>asdf</zorder>
      <zorder>label_8</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>TextMailFrom</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="zel.qrc"/>
 </resources>
 <connections/>
</ui>
